src/test/java/tutorspet/model/moduleclass/UniqueModuleClassListTest.java

```diff
@@ -143,8 +153,10 @@ public void removeUuid_existingUuid_updatesModuleClasses() {
     @Test
     public void removeUuid_nonExistingUuid_sameModuleClasses() {
         uniqueModuleClassList.add(CS2100_LAB);
-        UUID uuidToRemove = randomUUID();
-        uniqueModuleClassList.removeUuid(uuidToRemove);
+        Student randomStudentToRemove = new StudentBuilder().withUuid("4e13dcba-047d-4da7-9860-981493f1884e")
```

Comment:
Not sure if `invalidStudentToRemove` or `targetStudentToRemove` would sound better...